frontend/src/components/kanbanColumn/KanbanColumn.tsx

```diff
@@ -0,0 +1,97 @@
+import { Task } from '../../types/Task';
+import React from 'react';
+import { Draggable, Droppable } from 'react-beautiful-dnd';
+import { useEffect, useState } from 'react';
+import { MoreHoriz, CalendarMonth } from '@mui/icons-material';
+import './kanbanColumn.css';
+
+type KanbanColumnProps = {
+  tasks: Task[];
+  setTasks: React.Dispatch<React.SetStateAction<Task[]>>;
+  listId: string;
+};
+
+const KanbanColumn: React.FC<KanbanColumnProps> = ({ tasks, listId }) => {
+  const [enabled, setEnabled] = useState(false);
+  useEffect(() => {
+    const animation = requestAnimationFrame(() => setEnabled(true));
+    return () => {
+      cancelAnimationFrame(animation);
+      setEnabled(false);
+    };
+  }, []);
+  if (!enabled) {
+    return null;
+  }
+
+  return (
+    <div className='border-radius'>
+      <Droppable droppableId={listId} isDropDisabled={false}>
+        {(provided) => (
+          <div className='droppable' {...provided.droppableProps} ref={provided.innerRef}>
+            {tasks
+              .filter((task) => task.status === listId)
+              .map((task) => {
+                const globalIndex = tasks.findIndex((item) => item.id === task.id);
+                return (
+                  <Draggable key={task.id} draggableId={String(task.id)} index={globalIndex}>
+                    {(provided) => (
+                      <div
+                        className='kanban-task'
+                        key={task.id}
+                        ref={provided.innerRef}
+                        {...provided.draggableProps}
+                        {...provided.dragHandleProps}
+                      >
+                        <div className='kanban-task-row'>
+                          <span
+                            className={`priority ${
+                              task.priority === 'easy'
+                                ? 'green'
+                                : task.priority === 'hard'
+                                ? 'red'
+                                : task.priority === 'medium'
+                                ? 'orange'
+                                : null
```

Comment:
Similar comment i had in the other PR, let's not check what the value is, but use it directly. 
I would rename the css classes to easy, medium, hard or make a dictionary that maps between them